avm/res/compute/virtual-machine/main.bicep

```diff
@@ -131,6 +131,9 @@ param backupPolicyName string = 'DefaultPolicy'
 @description('Optional. The configuration for auto-shutdown.')
 param autoShutdownConfig object = {}
 
+@description('Optional. The resource Id of a maintenance configuration for this VM.')
+param maintenanceConfigurationId string = ''
```

Comment:
As per some spec that definitely exist, if this is a resource ID it should be in the name.

Reply:
fixed